src/nvim/ui_compositor.c

```diff
@@ -425,6 +425,15 @@ static void compose_line(Integer row, Integer startcol, Integer endcol,
     flags = flags & ~kLineFlagWrap;
   }
 
+  for (int i = skipstart; i < (endcol-skipend)-startcol; i++) {
+    if (attrbuf[i] < 0) {
+      if (rdb_flags & RDB_INVALID) {
+        abort();
+      } else {
+        attrbuf[i] = 0;
```

Comment:
btw: might have some logging for it?  Could be even `ILOG` I think (for visibility).